docs/design-system/preview/components.html

```
@@ -0,0 +1,116 @@
+    <span class="ovp-pill">running</span>
+    <span class="ovp-pill">stale</span>
+    <span class="ovp-pill">blocked</span>
+    <span class="ovp-pill">12 evidence</span>
+    <span class="ovp-pill">v0.2.0</span>
+  </div>
+
+  <div class="demo-block">
+    <div class="demo-label">FORM · input + button (primary) + button (secondary)</div>
+    <div style="display:flex;gap:10px;align-items:center;flex-wrap:wrap">
+      <input type="text" placeholder="Search the vault…" style="flex:1;min-width:240px">
+      <button>Search</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)